### PR TITLE
Add Dashboard API; Begin Integration of Configs

### PR DESCRIPTION
**Pull request by mcumbers**

### CODE REVIEW

File: src/listeners/guilds/members/guildMemberUpdate.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { initializeMember } from '#root/lib/util/functions/initialize';
```
Comment (rasmusgerdin) [inline src/listeners/guilds/members/guildMemberUpdate.ts:1]: Change to `#utils/functions/initialize`

File: src/listeners/guilds/messages/messageCreate.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { initializeMember } from '#root/lib/util/functions/initialize';
```
Comment (rasmusgerdin) [inline src/listeners/guilds/messages/messageCreate.ts:1]: Change to `#utils/functions/initialize`

File: src/listeners/ready.ts [hunk 3]
```diff
@@ -1,7 +1,8 @@
-import { CONTROL_GUILD, DEV, TOKENS, VERSION } from '#root/config';
+import { CONTROL_GUILD, DEV, INIT_ALL_MEMBERS, INIT_ALL_USERS, TOKENS, VERSION } from '#root/config';
+import { initializeGuild, initializeMember, initializeUser } from '#root/lib/util/functions/initialize';
```
Comment (rasmusgerdin) [inline src/listeners/ready.ts:2]: Change to `#utils/functions/initialize`

**Review by noragerdin — DISMISSED** [review]
Looks great!

**Review by noragerdin — APPROVED** [review]

**Pull request merged**